node/manifest.ts

```diff
@@ -54,14 +65,29 @@ const sanitizeEdgeFunctionConfig = (config: Record<string, EdgeFunctionConfig>):
   const newConfig: Record<string, EdgeFunctionConfig> = {}
 
   for (const [name, functionConfig] of Object.entries(config)) {
-    if (functionConfig.excluded_patterns.length !== 0 || functionConfig.on_error) {
-      newConfig[name] = functionConfig
+    const newFunctionConfig = removeEmptyConfigValues(functionConfig)
+
+    if (hasAnyConfigValues(newFunctionConfig)) {
```

Comment:
Wouldn't this be the same as `Object.keys(newFunctionConfig).length > 0`?

Reply:
For some reason, before that didn't work... but now it does! Nice catch!